src/index.html

```diff
@@ -0,0 +1,369 @@
+            Somos livres de astericos & tarifas escondidas.
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section id="card" class="card">
+      <div class="container">
+        <div class="card__content">
+          <h1 class="title">
+            Novo <span class="marked">CodeCard</span>. 💳 <span class="non-highlighted">Use débito ou crédito, você é
+              quem sabe</span>
+          </h1>
+
+          <p class="subtitle non-highlighted">
+            Faça parte da Codebank solicitando agora sua Conta
+          </p>
+        </div>
+
+        <div class="card__image">
+          <img src="./img/card.svg" />
+        </div>
+
+        <div class="card__contactless">
+          <img class="card__contactless__image" src="./img/contactless.svg" />
+          <p class="subtitle">
+            Adicione seu cartão na Wallet. Apenas toque e pague rápido com o contactless
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section class="codeline">
+      <div class="container">
+        <div class="codeline__content">
+          <h1 class="title">
+            Faça pagamentos com <span class="marked">uma linha de código</span>. ✨
+            <span class="non-highlighted">Simples.</span>
+          </h1>
+
+          <p class="subtitle non-highlighted">
+            Faça parte da Codebank solicitando agora sua Conta
+          </p>
+        </div>
+
+        <div class="border border--gray border--termynal">
+          <div class="codeline__border__termynal" data-termynal data-ty-typeDelay="70" data-ty-lineDelay="900">
+            <span data-ty="input" data-ty-prompt="$">cb pay 13691402750500290410887318880015982870000012000</span>
+            <span data-ty>Você deseja pagar o boleto da Aeogi, no valor de R$ 120,00?</span>
+            <span data-ty="input" data-ty-typeDelay="1000" data-ty-prompt="(s/N)">s</span>
+            <span data-ty="progress" data-ty-progressChar="·"></span>
+            <span data-ty>Pagamento efetuado com sucesso!</span>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section id="api" class="api">
+      <div class="container">
+        <div class="api__content">
+          <h1 class="title">
+            API de <span class="marked">Ted</span> e <span class="marked">Boleto</span>. 💸 📑
+            <span class="non-highlighted"> Pague e faça emissão e recebimento de TED's.</span>
+          </h1>
+
+          <p class="subtitle non-highlighted">
+            Faça parte da Codebank solicitando agora sua Conta
+          </p>
+        </div>
+
+          <div class="container--api__code">
+              <div class="api__code">
+                <pre>
+                  <div class="border border--gray border--api__code">
+                    <code class="javascript">// Faz o require da lib do Codebank e inicia com uma chave de teste
+                      const cb = require('@codebank/ted')('test_ghvbityg9086tftuhkgvtr8r7ityghuf');
```

Comment:
O código está dando scroll horizontal, acho melhor colocar assim...

```suggestion
                      const API_KEY = 'test_ghvbityg9086tftuhkgvtr8r7ityghuf';
                      const cb = require('@codebank/ted')(API_KEY);
```